Scripts/Build-Phases/copy-secret.sh

```diff
@@ -59,6 +59,11 @@ ensure_is_in_input_files_list $EXAMPLE_SECRETS_FILE
 SECRETS_DESTINATION_FILE="${SRCROOT}/Simplenote/Credentials/SPCredentials.swift"
 mkdir -p $(dirname "$SECRETS_DESTINATION_FILE")
 
+if cmp --silent -- ${SECRETS_FILE} ${SECRETS_DESTINATION_FILE}; then
```

Comment:
I am not familiar with `cmp` but I wonder if we should do this after checking the existence of the `SECRETS_FILE` and `SECRETS_DESTINATION_FILE`?

Reply:
If the files aren't equal -or don't exist, AFAIK- this would evaluate false, soooo we get away without modifying the rest of the script.

Hi there sir!! thanks for checking!!